<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать этикеток</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditL">
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="cmdPrint">
      <property name="text">
       <string>Создать этикетки</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editPart"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="editMas">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="boxDiam"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editPlav"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Масса нетто, кг</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Плавка</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Партия</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Диаметр</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Марка</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Длина, мм</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="boxMarka"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cmdUpd">
      <property name="text">
       <string>Обновить данные</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Дата</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>2</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QListView" name="listViewGost"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Документация</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>boxMarka</tabstop>
  <tabstop>boxDiam</tabstop>
  <tabstop>editPlav</tabstop>
  <tabstop>editPart</tabstop>
  <tabstop>lineEditL</tabstop>
  <tabstop>editMas</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>listViewGost</tabstop>
  <tabstop>cmdPrint</tabstop>
  <tabstop>cmdUpd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
